pyopae/opae.cpp

```diff
@@ -164,7 +164,7 @@ PYBIND11_MODULE(_opae, m) {
            py::arg("offset"), py::arg("value"), py::arg("csr_space") = 0);
 
   // define shared_buffer class
-  m.def("allocate_shared_buffer", &shared_buffer::allocate,
+  m.def("allocate_shared_buffer", shared_buffer_allocate,
```

Comment:
What is the difference between the original and the modified here?

Reply:
The original refers to a static method defined in the `opae::fpga::types::shared_buffer` class.
The new one refers to a function defined in pyshared_buffer.h|cpp which does a little more that simply calling `shared_buffer::allocate`